File: rest/recording/instance-delete-examples/instance-delete-examples.5.x.php
<?php
// Get the PHP helper library from twilio.com/docs/php/install
require_once '/path/to/vendor/autoload.php'; // Loads the library
use \Twilio\Rest\Client;

// Your Account Sid and Auth Token from twilio.com/user/account
$sid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
$token = "your_auth_token";
$client = new Client($sid, $token);

$client->account
    ->recordings("RE557ce644e5ab84fa21cc21112e22c485")
    ->delete();

File: rest/recording/instance-delete-examples/instance-delete-examples.5.x.rb
require 'twilio-ruby'

# put your own credentials here
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'

# set up a client to talk to the Twilio REST API
@client = Twilio::REST::Client.new(account_sid, auth_token)

@recording = @client.api
                    .recordings('RE557ce644e5ab84fa21cc21112e22c485').fetch

@recording.delete
